plugin/db/pg/pg.go

```diff
@@ -1651,10 +1664,7 @@ func getIndexColumnExpressions(stmt string) ([]string, error) {
 
 	var cols []string
 	re := regexp.MustCompile(`\(\(.*\)\)`)
-	for {
-		if len(columnStmt) <= 0 {
-			break
-		}
+	for len(columnStmt) > 0 {
```

Comment:
Leave this without a change?

Reply:
lint hinted that it could be optimised so it was changed in the process

Reply:
I'll submit a separate PR later, which is indeed unrelated to this change